pages/index.vue

```diff
@@ -53,26 +53,26 @@ export default {
   }),
   computed: {
     isLoading() {
-      return this.$store.loading;
```

Comment:
Why return the authToken instead of loading? 
This computed property is used to disable the login button until the api request returns. Am I going about it wrong?